### ISSUE TITLE
Application does not work in microsoft-edge-legacy

### ISSUE THREAD

**Issue by sebastianzillessen** [Zuehlke/poinz#147]
In out project we are using microsoft-edge-legacy and cannot use poinz as the JS does not work. Might just be a problem of the target of the ECMA Script Target Version?

Console Output:
```
00: Navigation occurred.
poinz.herokuapp.com (1,1)
0: Expected identifier, string or number
bundle.js (1,370148)
```

**Comment by xeronimus** [comment]
@sebastianzillessen I pushed a fix to branch 147.  I tested it locally with "Microsoft Edge 41.16299.1480.0  /Microsoft EdgeHTML 16.16299"  on Windows 10

**Comment by xeronimus** [comment]
@sebastianzillessen   this is now merged and deployed (v 0.8.5)  . Could you check it out?

**Comment by sebastianzillessen** [comment]
Works great now! Thanks!
